tests/devices/unit_tests/test_positioner.py

```diff
@@ -1,27 +1,36 @@
 import pytest
-from ophyd_async.core import init_devices
+from ophyd_async.core import StrictEnum, init_devices
 from ophyd_async.testing import (
     get_mock_put,
 )
 
-from dodal.devices.i10.diagnostics import D3Position, D5APosition, Positioner
+from dodal.devices.positioner import Positioner1D, create_positioner
+
+
+class Positions(StrictEnum):
+    TEST_1 = "Test"
+    TEST_2 = "Test_2"
+
+
+class BadPositions(StrictEnum):
+    BAD_POSITION = "BAD"
 
 
 @pytest.fixture
-async def mock_positioner(prefix: str = "BLXX-EA-007:") -> Positioner:
+async def mock_positioner(prefix: str = "BLXX-EA-007:") -> Positioner1D:
```

Comment:
```suggestion
async def mock_positioner() -> Positioner1D:
```

nit: the default prefix is never overridden